[2023-01-21T17:04:35.212+0000] {processor.py:153} INFO - Started process (PID=19624) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:04:35.213+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:04:35.214+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:04:35.214+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:04:35.226+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:04:35.224+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:04:35.226+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:04:35.249+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.040 seconds
[2023-01-21T17:05:05.400+0000] {processor.py:153} INFO - Started process (PID=19668) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:05:05.401+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:05:05.402+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:05:05.402+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:05:05.407+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:05:05.406+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:05:05.407+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:05:05.431+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.036 seconds
[2023-01-21T17:05:28.562+0000] {processor.py:153} INFO - Started process (PID=19695) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:05:28.564+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:05:28.565+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:05:28.564+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:05:28.571+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:05:28.570+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:05:28.571+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:05:28.594+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.035 seconds
[2023-01-21T17:05:58.761+0000] {processor.py:153} INFO - Started process (PID=19746) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:05:58.762+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:05:58.763+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:05:58.763+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:05:58.768+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:05:58.767+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:05:58.769+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:05:58.792+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.035 seconds
[2023-01-21T17:06:28.963+0000] {processor.py:153} INFO - Started process (PID=19781) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:06:28.964+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:06:28.965+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:06:28.965+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:06:28.970+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:06:28.969+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:06:28.970+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:06:29.034+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.074 seconds
[2023-01-21T17:06:59.146+0000] {processor.py:153} INFO - Started process (PID=19816) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:06:59.147+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:06:59.148+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:06:59.148+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:06:59.152+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:06:59.152+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:06:59.153+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:06:59.176+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.034 seconds
[2023-01-21T17:07:29.367+0000] {processor.py:153} INFO - Started process (PID=19851) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:07:29.368+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:07:29.369+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:07:29.368+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:07:29.374+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:07:29.373+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:07:29.375+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:07:29.399+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.036 seconds
[2023-01-21T17:07:59.630+0000] {processor.py:153} INFO - Started process (PID=19902) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:07:59.632+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:07:59.633+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:07:59.633+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:07:59.641+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:07:59.640+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:07:59.642+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:07:59.677+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.052 seconds
[2023-01-21T17:08:29.891+0000] {processor.py:153} INFO - Started process (PID=19937) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:08:29.892+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:08:29.893+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:08:29.893+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:08:29.898+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:08:29.897+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:08:29.898+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:08:29.922+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.037 seconds
[2023-01-21T17:09:00.090+0000] {processor.py:153} INFO - Started process (PID=19972) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:09:00.091+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:09:00.092+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:09:00.092+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:09:00.097+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:09:00.096+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:09:00.097+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:09:00.120+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.034 seconds
[2023-01-21T17:09:30.279+0000] {processor.py:153} INFO - Started process (PID=20023) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:09:30.280+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:09:30.282+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:09:30.281+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:09:30.288+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:09:30.287+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:09:30.289+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:09:30.319+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.046 seconds
[2023-01-21T17:10:00.499+0000] {processor.py:153} INFO - Started process (PID=20058) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:10:00.500+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:10:00.501+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:10:00.501+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:10:00.507+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:10:00.506+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:10:00.508+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:10:00.538+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.043 seconds
[2023-01-21T17:10:30.685+0000] {processor.py:153} INFO - Started process (PID=20093) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:10:30.686+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:10:30.687+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:10:30.687+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:10:30.695+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:10:30.694+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:10:30.695+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:10:30.719+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.038 seconds
[2023-01-21T17:11:00.884+0000] {processor.py:153} INFO - Started process (PID=20139) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:11:00.886+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:11:00.886+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:11:00.886+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:11:00.892+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:11:00.891+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:11:00.892+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:11:00.915+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.034 seconds
[2023-01-21T17:11:31.073+0000] {processor.py:153} INFO - Started process (PID=20181) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:11:31.075+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:11:31.076+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:11:31.075+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:11:31.080+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:11:31.080+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:11:31.081+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:11:31.104+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.035 seconds
[2023-01-21T17:12:01.264+0000] {processor.py:153} INFO - Started process (PID=20216) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:12:01.265+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:12:01.266+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:12:01.266+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:12:01.270+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:12:01.270+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:12:01.271+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:12:01.294+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.033 seconds
[2023-01-21T17:12:31.458+0000] {processor.py:153} INFO - Started process (PID=20251) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:12:31.460+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:12:31.461+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:12:31.461+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:12:31.468+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:12:31.467+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:12:31.469+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:12:31.494+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.043 seconds
[2023-01-21T17:13:01.643+0000] {processor.py:153} INFO - Started process (PID=20302) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:13:01.645+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:13:01.646+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:13:01.646+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:13:01.651+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:13:01.651+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:13:01.652+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:13:01.676+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.036 seconds
[2023-01-21T17:13:31.841+0000] {processor.py:153} INFO - Started process (PID=20337) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:13:31.842+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:13:31.843+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:13:31.843+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:13:31.848+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:13:31.847+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:13:31.848+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:13:31.873+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.036 seconds
[2023-01-21T17:14:02.041+0000] {processor.py:153} INFO - Started process (PID=20372) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:14:02.042+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:14:02.043+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:14:02.043+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:14:02.048+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:14:02.047+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:14:02.048+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:14:02.072+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.034 seconds
[2023-01-21T17:14:32.233+0000] {processor.py:153} INFO - Started process (PID=20423) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:14:32.234+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:14:32.235+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:14:32.235+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:14:32.241+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:14:32.240+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:14:32.241+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:14:32.267+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.038 seconds
[2023-01-21T17:15:02.465+0000] {processor.py:153} INFO - Started process (PID=20458) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:15:02.466+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:15:02.466+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:15:02.466+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:15:02.472+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:15:02.471+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:15:02.473+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:15:02.500+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.041 seconds
[2023-01-21T17:15:32.684+0000] {processor.py:153} INFO - Started process (PID=20493) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:15:32.686+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:15:32.686+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:15:32.686+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:15:32.692+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:15:32.690+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:15:32.692+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:15:32.716+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.035 seconds
[2023-01-21T17:16:02.885+0000] {processor.py:153} INFO - Started process (PID=20537) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:16:02.886+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:16:02.886+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:16:02.886+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:16:02.892+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:16:02.891+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:16:02.892+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:16:02.918+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.037 seconds
[2023-01-21T17:16:33.093+0000] {processor.py:153} INFO - Started process (PID=20579) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:16:33.094+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:16:33.095+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:16:33.095+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:16:33.101+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:16:33.100+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:16:33.101+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:16:33.126+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.037 seconds
[2023-01-21T17:17:03.300+0000] {processor.py:153} INFO - Started process (PID=20614) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:17:03.301+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:17:03.302+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:17:03.302+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:17:03.307+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:17:03.306+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:17:03.307+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:17:03.330+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.034 seconds
[2023-01-21T17:17:33.494+0000] {processor.py:153} INFO - Started process (PID=20649) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:17:33.496+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:17:33.497+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:17:33.497+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:17:33.503+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:17:33.502+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:17:33.503+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:17:33.528+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.039 seconds
[2023-01-21T17:18:03.700+0000] {processor.py:153} INFO - Started process (PID=20700) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:18:03.702+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:18:03.703+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:18:03.702+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:18:03.708+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:18:03.707+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:18:03.708+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:18:03.737+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.043 seconds
[2023-01-21T17:18:33.882+0000] {processor.py:153} INFO - Started process (PID=20735) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:18:33.884+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:18:33.885+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:18:33.885+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:18:33.891+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:18:33.890+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:18:33.891+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:18:33.916+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.038 seconds
[2023-01-21T17:19:04.082+0000] {processor.py:153} INFO - Started process (PID=20770) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:19:04.082+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:19:04.084+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:19:04.083+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:19:04.090+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:19:04.089+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:19:04.091+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:19:04.114+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.036 seconds
[2023-01-21T17:19:34.286+0000] {processor.py:153} INFO - Started process (PID=20821) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:19:34.288+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:19:34.289+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:19:34.289+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:19:34.296+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:19:34.295+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:19:34.297+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:19:34.325+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.043 seconds
[2023-01-21T17:20:04.498+0000] {processor.py:153} INFO - Started process (PID=20856) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:20:04.499+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:20:04.500+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:20:04.499+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:20:04.506+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:20:04.504+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:20:04.507+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:20:04.532+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.038 seconds
[2023-01-21T17:20:34.699+0000] {processor.py:153} INFO - Started process (PID=20891) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:20:34.700+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:20:34.701+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:20:34.701+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:20:34.707+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:20:34.706+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:20:34.707+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:20:34.735+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.039 seconds
[2023-01-21T17:21:04.883+0000] {processor.py:153} INFO - Started process (PID=20935) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:21:04.884+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:21:04.885+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:21:04.885+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:21:04.890+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:21:04.889+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:21:04.890+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:21:04.914+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.035 seconds
[2023-01-21T17:21:35.077+0000] {processor.py:153} INFO - Started process (PID=20978) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:21:35.078+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:21:35.079+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:21:35.079+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:21:35.084+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:21:35.083+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:21:35.085+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:21:35.111+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.038 seconds
[2023-01-21T17:22:05.274+0000] {processor.py:153} INFO - Started process (PID=21013) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:22:05.275+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:22:05.276+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:22:05.275+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:22:05.280+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:22:05.279+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:22:05.280+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:22:05.304+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.033 seconds
[2023-01-21T17:22:35.479+0000] {processor.py:153} INFO - Started process (PID=21048) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:22:35.480+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:22:35.481+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:22:35.481+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:22:35.487+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:22:35.486+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:22:35.487+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:22:35.511+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.037 seconds
[2023-01-21T17:23:05.680+0000] {processor.py:153} INFO - Started process (PID=21099) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:23:05.681+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:23:05.682+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:23:05.682+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:23:05.688+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:23:05.687+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:23:05.689+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:23:05.717+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.043 seconds
[2023-01-21T17:23:35.872+0000] {processor.py:153} INFO - Started process (PID=21134) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:23:35.873+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:23:35.875+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:23:35.875+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:23:35.880+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:23:35.879+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:23:35.881+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:23:35.907+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.039 seconds
[2023-01-21T17:24:06.082+0000] {processor.py:153} INFO - Started process (PID=21169) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:24:06.083+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:24:06.084+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:24:06.084+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:24:06.090+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:24:06.089+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:24:06.090+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:24:06.113+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.036 seconds
[2023-01-21T17:24:36.276+0000] {processor.py:153} INFO - Started process (PID=21213) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:24:36.278+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T17:24:36.279+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:24:36.279+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:24:36.286+0000] {logging_mixin.py:137} INFO - [2023-01-21T17:24:36.285+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 2, in <module>
    from airflow.contrib.operators.hdfs_operator import HDFSOperator
ModuleNotFoundError: No module named 'airflow.contrib.operators.hdfs_operator'
[2023-01-21T17:24:36.287+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T17:24:36.374+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.101 seconds
[2023-01-21T18:32:00.725+0000] {processor.py:153} INFO - Started process (PID=54) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:32:00.728+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:32:00.729+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:32:00.729+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:32:00.752+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:32:00.751+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 4, in <module>
    from hdfs import InsecureClient
ModuleNotFoundError: No module named 'hdfs'
[2023-01-21T18:32:00.752+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:32:00.783+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.064 seconds
[2023-01-21T18:32:30.935+0000] {processor.py:153} INFO - Started process (PID=89) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:32:30.936+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:32:30.936+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:32:30.936+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:32:30.944+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:32:30.943+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 4, in <module>
    from hdfs import InsecureClient
ModuleNotFoundError: No module named 'hdfs'
[2023-01-21T18:32:30.944+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:32:30.970+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.039 seconds
[2023-01-21T18:33:01.133+0000] {processor.py:153} INFO - Started process (PID=125) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:33:01.135+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:33:01.135+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:33:01.135+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:33:01.143+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:33:01.142+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 4, in <module>
    from hdfs import InsecureClient
ModuleNotFoundError: No module named 'hdfs'
[2023-01-21T18:33:01.143+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:33:01.171+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.041 seconds
[2023-01-21T18:33:31.363+0000] {processor.py:153} INFO - Started process (PID=169) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:33:31.372+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:33:31.374+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:33:31.373+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:33:31.388+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:33:31.387+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 4, in <module>
    from hdfs import InsecureClient
ModuleNotFoundError: No module named 'hdfs'
[2023-01-21T18:33:31.390+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:33:31.420+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.062 seconds
[2023-01-21T18:34:01.595+0000] {processor.py:153} INFO - Started process (PID=211) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:34:01.596+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:34:01.597+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:34:01.596+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:34:01.605+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:34:01.604+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 4, in <module>
    from hdfs import InsecureClient
ModuleNotFoundError: No module named 'hdfs'
[2023-01-21T18:34:01.605+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:34:01.631+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.040 seconds
[2023-01-21T18:34:31.790+0000] {processor.py:153} INFO - Started process (PID=246) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:34:31.791+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:34:31.792+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:34:31.792+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:34:31.800+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:34:31.799+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 4, in <module>
    from hdfs import InsecureClient
ModuleNotFoundError: No module named 'hdfs'
[2023-01-21T18:34:31.801+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:34:31.827+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.041 seconds
[2023-01-21T18:35:01.998+0000] {processor.py:153} INFO - Started process (PID=288) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:35:01.999+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:35:02.000+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:35:02.000+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:35:02.007+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:35:02.006+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/hdfs_first_version.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hdfs_first_version.py", line 4, in <module>
    from hdfs import InsecureClient
ModuleNotFoundError: No module named 'hdfs'
[2023-01-21T18:35:02.007+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:35:02.033+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.041 seconds
[2023-01-21T18:36:25.100+0000] {processor.py:153} INFO - Started process (PID=66) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:36:25.103+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:36:25.104+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:25.104+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:36:25.172+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:36:25.468+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:25.468+0000] {manager.py:504} INFO - Created Permission View: can delete on DAG:hdfs_files_dag
[2023-01-21T18:36:25.484+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:25.484+0000] {manager.py:504} INFO - Created Permission View: can edit on DAG:hdfs_files_dag
[2023-01-21T18:36:25.501+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:25.501+0000] {manager.py:504} INFO - Created Permission View: can read on DAG:hdfs_files_dag
[2023-01-21T18:36:25.502+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:25.501+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:36:25.520+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:25.520+0000] {dag.py:2697} INFO - Creating ORM DAG for hdfs_files_dag
[2023-01-21T18:36:25.534+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:25.534+0000] {dag.py:3427} INFO - Setting next_dagrun for hdfs_files_dag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2023-01-21T18:36:25.561+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.467 seconds
[2023-01-21T18:36:55.815+0000] {processor.py:153} INFO - Started process (PID=119) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:36:55.818+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:36:55.830+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:55.830+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:36:55.973+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:36:56.126+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:56.126+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:36:56.203+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:36:56.203+0000] {dag.py:3427} INFO - Setting next_dagrun for hdfs_files_dag to 2022-01-07T00:00:00+00:00, run_after=2022-01-08T00:00:00+00:00
[2023-01-21T18:36:56.251+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.441 seconds
[2023-01-21T18:37:26.406+0000] {processor.py:153} INFO - Started process (PID=154) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:37:26.407+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:37:26.407+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:37:26.407+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:37:26.437+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:37:26.494+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:37:26.493+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:37:26.562+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.160 seconds
[2023-01-21T18:37:56.611+0000] {processor.py:153} INFO - Started process (PID=190) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:37:56.612+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:37:56.614+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:37:56.614+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:37:56.639+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:37:56.689+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:37:56.688+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:37:56.749+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.142 seconds
[2023-01-21T18:38:26.882+0000] {processor.py:153} INFO - Started process (PID=223) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:38:26.884+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:38:26.885+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:38:26.884+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:38:26.909+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:38:26.961+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:38:26.961+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:38:27.034+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.157 seconds
[2023-01-21T18:38:57.200+0000] {processor.py:153} INFO - Started process (PID=267) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:38:57.201+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:38:57.202+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:38:57.202+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:38:57.233+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:38:57.292+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:38:57.292+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:38:57.367+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.173 seconds
[2023-01-21T18:39:27.549+0000] {processor.py:153} INFO - Started process (PID=309) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:39:27.551+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:39:27.552+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:39:27.551+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:39:27.586+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:39:27.652+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:39:27.651+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:39:27.723+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.179 seconds
[2023-01-21T18:39:58.401+0000] {processor.py:153} INFO - Started process (PID=344) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:39:58.402+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:39:58.403+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:39:58.403+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:39:58.438+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:39:58.508+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:39:58.507+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:39:58.586+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.190 seconds
[2023-01-21T18:40:28.899+0000] {processor.py:153} INFO - Started process (PID=379) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:40:28.900+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:40:28.901+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:40:28.900+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:40:28.928+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:40:28.981+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:40:28.981+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:40:29.057+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.165 seconds
[2023-01-21T18:40:59.228+0000] {processor.py:153} INFO - Started process (PID=414) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:40:59.230+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:40:59.231+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:40:59.231+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:40:59.257+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:40:59.309+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:40:59.309+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:40:59.374+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.151 seconds
[2023-01-21T18:41:29.571+0000] {processor.py:153} INFO - Started process (PID=458) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:41:29.572+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:41:29.573+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:41:29.573+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:41:29.605+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:41:29.738+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:41:29.738+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:41:29.816+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.249 seconds
[2023-01-21T18:42:00.006+0000] {processor.py:153} INFO - Started process (PID=512) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:42:00.045+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:42:00.046+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:42:00.046+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:42:00.164+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:42:00.319+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:42:00.319+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:42:00.453+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:42:00.452+0000] {dag.py:3427} INFO - Setting next_dagrun for hdfs_files_dag to 2022-01-17T00:00:00+00:00, run_after=2022-01-18T00:00:00+00:00
[2023-01-21T18:42:00.522+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.522 seconds
[2023-01-21T18:42:30.767+0000] {processor.py:153} INFO - Started process (PID=559) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:42:30.768+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:42:30.769+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:42:30.769+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:42:30.798+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:42:30.854+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:42:30.854+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:42:30.922+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.158 seconds
[2023-01-21T18:43:01.090+0000] {processor.py:153} INFO - Started process (PID=594) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:43:01.091+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:43:01.092+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:43:01.092+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:43:01.119+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:43:01.170+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:43:01.169+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:43:01.245+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.159 seconds
[2023-01-21T18:43:31.432+0000] {processor.py:153} INFO - Started process (PID=629) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:43:31.433+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:43:31.434+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:43:31.433+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:43:31.464+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:43:31.526+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:43:31.526+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:43:31.604+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.177 seconds
[2023-01-21T18:44:01.799+0000] {processor.py:153} INFO - Started process (PID=680) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:44:01.801+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:44:01.802+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:44:01.802+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:44:01.833+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:44:01.894+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:44:01.893+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:44:01.967+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.172 seconds
[2023-01-21T18:44:32.156+0000] {processor.py:153} INFO - Started process (PID=716) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:44:32.157+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:44:32.159+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:44:32.158+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:44:32.192+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:44:32.252+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:44:32.251+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:44:32.339+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.187 seconds
[2023-01-21T18:45:02.556+0000] {processor.py:153} INFO - Started process (PID=751) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:45:02.557+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:45:02.560+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:45:02.559+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:45:02.587+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:45:02.648+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:45:02.648+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:45:02.714+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.162 seconds
[2023-01-21T18:45:32.895+0000] {processor.py:153} INFO - Started process (PID=786) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:45:32.896+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:45:32.897+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:45:32.897+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:45:32.925+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:45:32.978+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:45:32.978+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:45:33.046+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.156 seconds
[2023-01-21T18:46:03.221+0000] {processor.py:153} INFO - Started process (PID=837) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:46:03.222+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:46:03.223+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:46:03.223+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:46:03.253+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:46:03.301+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:46:03.301+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:46:03.374+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.157 seconds
[2023-01-21T18:46:33.554+0000] {processor.py:153} INFO - Started process (PID=873) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:46:33.555+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:46:33.555+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:46:33.555+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:46:33.589+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:46:33.651+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:46:33.650+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:46:33.755+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.205 seconds
[2023-01-21T18:47:03.989+0000] {processor.py:153} INFO - Started process (PID=908) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:47:03.991+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:47:03.993+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:47:03.993+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:47:04.033+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:47:04.092+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:47:04.092+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:47:04.170+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.186 seconds
[2023-01-21T18:47:34.429+0000] {processor.py:153} INFO - Started process (PID=965) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:47:34.430+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:47:34.431+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:47:34.431+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:47:34.468+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:47:34.528+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:47:34.528+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:47:34.583+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:47:34.582+0000] {dag.py:3427} INFO - Setting next_dagrun for hdfs_files_dag to 2022-02-02T00:00:00+00:00, run_after=2022-02-03T00:00:00+00:00
[2023-01-21T18:47:34.618+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.195 seconds
[2023-01-21T18:48:05.352+0000] {processor.py:153} INFO - Started process (PID=1000) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:48:05.353+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:48:05.354+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:48:05.354+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:48:05.380+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:48:05.427+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:48:05.427+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:48:05.493+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.146 seconds
[2023-01-21T18:48:35.665+0000] {processor.py:153} INFO - Started process (PID=1035) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:48:35.667+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:48:35.667+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:48:35.667+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:48:35.701+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:48:35.757+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:48:35.757+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:48:35.831+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.171 seconds
[2023-01-21T18:49:06.001+0000] {processor.py:153} INFO - Started process (PID=1087) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:49:06.003+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:49:06.003+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:49:06.003+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:49:06.034+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:49:06.088+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:49:06.088+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:49:06.150+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.154 seconds
[2023-01-21T18:49:36.313+0000] {processor.py:153} INFO - Started process (PID=1122) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:49:36.315+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:49:36.316+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:49:36.315+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:49:36.345+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:49:36.397+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:49:36.397+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:49:36.463+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.154 seconds
[2023-01-21T18:50:06.637+0000] {processor.py:153} INFO - Started process (PID=1157) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:50:06.639+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:50:06.641+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:50:06.640+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:50:06.667+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:50:06.721+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:50:06.721+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:50:06.786+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.153 seconds
[2023-01-21T18:50:36.967+0000] {processor.py:153} INFO - Started process (PID=1201) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:50:36.968+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:50:36.969+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:50:36.969+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:50:37.002+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:50:37.060+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:50:37.060+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:50:37.128+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.165 seconds
[2023-01-21T18:51:07.303+0000] {processor.py:153} INFO - Started process (PID=1243) to work on /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:51:07.305+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/hdfs_first_version.py for tasks to queue
[2023-01-21T18:51:07.306+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:51:07.306+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:51:07.332+0000] {processor.py:753} INFO - DAG(s) dict_keys(['hdfs_files_dag']) retrieved from /opt/airflow/dags/hdfs_first_version.py
[2023-01-21T18:51:07.388+0000] {logging_mixin.py:137} INFO - [2023-01-21T18:51:07.387+0000] {dag.py:2676} INFO - Sync 1 DAGs
[2023-01-21T18:51:07.459+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/hdfs_first_version.py took 0.160 seconds
